### PR TITLE
Add security headers and disable nginx version exposure

### PR DESCRIPTION
**Pull request by degeri**
Edited Dockerfile to copy over conf/nginx.conf 

New nginx.conf contains the following. 

Adds 
"server_tokens off;" which disables nginx version from being shown. 

Adds the following security headers.

add_header X-Content-Type-Options "nosniff" always;  ==> Prevents "mime" based attacks. 
add_header X-Frame-Options DENY always;   ==> Disable iframe of the website. 
add_header X-XSS-Protection "1; mode=block" always;  ==> Enable XSS protection. 
add_header Referrer-Policy "no-referrer" always; ==> Disable referrer from being shared on any request.

"always" forces these headers for all response codes.

http://nginx.org/en/docs/http/ngx_http_headers_module.html

### CODE REVIEW

File: conf/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,53 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    #Disable nginx
```
Comment (dajohi) [inline conf/nginx.conf]: `# Hide version`

File: conf/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,53 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    #Disable nginx 
+    server_tokens off;
+    #Security Headers
```
Comment (dajohi) [inline conf/nginx.conf]: new line since new comment

File: conf/nginx.conf [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    #error_page  404              /404.html;
+
+    # redirect server error pages to the static page /50x.html
+    #
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
```
Comment (dajohi) [inline conf/nginx.conf]: remove the commented out section below.

File: conf/nginx.conf (hunk 1 above)
Comment (degeri) [reply conf/nginx.conf]: Done

File: conf/nginx.conf (hunk 2 above)
Comment (degeri) [reply conf/nginx.conf]: Done

File: conf/nginx.conf (hunk 3 above)
Comment (degeri) [reply conf/nginx.conf]: Done

**Pull request merged**